### PR TITLE
feat: instruction hiding with button

### LINKED ISSUE #45950: [multifile editor: cert projects] Add a button to hide instructions
### Describe the Issue

When working on the RWD cert projects using the multifile editor, space is somewhat limited. A button to toggle the instructions might be nice.

### Steps to Reproduce

1. Go to https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-tribute-page-project/build-a-tribute-page
2. See instructions
3. No way to hide them (edit: I guess you can change the width, but I don't think that's as good of a UX)

### Expected behavior

A way to toggle instructions to give more space

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #45950

### CODE REVIEW

**Comment by nhcarrigan** [comment]
image


I wonder if we should put the instructions button on the left, since the instructions panel is on the left? 🤔

**Comment by nhcarrigan** [comment]
image


I wonder if we should put the instructions button on the left, since the instructions panel is on the left? 🤔

**Comment by bbsmooth** [comment]
Hey @Sembauke, if you look at the other toggle buttons in the action row you'll see that they have the `aria-expanded` property set on them, to either `true` if their contents are showing, or `false` if their contents are hidden. We'll need to do the same for this button as well.

**Comment by Sembauke** [comment]
@bbsmooth  Thank you! I completely looked over that. Thanks for giving accessibility tips.

**Comment by moT01** [comment]
I would vote to put it on the left @nhcarrigan

**Comment by Sembauke** [comment]
Yes in my opinion it would make more sense to keep it on the left, as the panel is also on the left.

**Comment by ahmadabdolsaheb** [comment]
Looking good. For now the button could go to the left of the editor tabs.

**Comment by Sembauke** [comment]
Thank you for confirming Ahmad.

**Comment by ahmadabdolsaheb** [comment]
Oh I meant left 🤦‍♂️. I need to learn the directions at some point. :)
We will be grouping all of those buttons based on their functionality (toggling panes) and displaying them based on the order of their corresponding panes. But for now just placing the instructions button on the left side of the action row should do the trick.

**Comment by Sembauke** [comment]
You mean right of the editor tabs right. :)

**Comment by ahmadabdolsaheb** [comment]
No left of the editor tabs. Just to be clear here is a quick wireframe :)
Screen Shot 2022-05-25 at 2 39 35 PM

**Comment by Sembauke** [comment]
@ahmadabdolsaheb Should be good to go.

**Review by moT01 — DISMISSED** [review]
LGTM @Sembauke 🎉 Everything is working real good. The only thing that we could maybe add here is some cypress tests to check if it's showing/not showing on the correct pages and that it correctly toggles the pane. I would be fine with it going in like this, though. Maybe see what others think.

File: client/src/templates/Challenges/components/tool-panel.css [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 .tool-panel-group .dropdown-menu {
   top: auto;
-  bottom: 100%;
+  bottom: 0;
```
Comment (ahmadabdolsaheb) [inline client/src/templates/Challenges/components/tool-panel.css:40]:
```suggestion
  bottom: 100%;
```

**Review by ahmaxed — APPROVED** [review]

**Review by raisedadead — APPROVED** [review]

**Pull request merged**